components/micro-gateway-core/src/main/ballerina/src/gateway/grpc_client/AnalyticsSendService.bal

```diff
@@ -0,0 +1,177 @@
+import ballerina/grpc;
```

Comment:
All our bal files are named with `_` naming. Rename the file with underscore naming. Also I think this name is not meaningful enough. Shall we rename? @Rajith90 @hasuniea @VirajSalaka WDYT?